nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -634,6 +634,8 @@
 
 - `iproute2` now has libbpf support.
 
+- `xrootd` version `5.7.1` no longer produces `bin/test-runner` for test execution. Therefore, the `enableTestRunner` argument are removed from the override interface, and so does the `xrootd.tests.test-runner` package test.
```

Comment:
Was this something externally exposed? Sounds more like an internal option to me.

Reply:
Normally we don't really care much about the override options, especially if it was an internal one.

Reply:
Removed.